changelogs/unreleased/gh-9195-not-log-unchanged-box-cfg-options.md

```diff
@@ -0,0 +1,3 @@
+## feature/core
+
+* The option value is no longer logged if it has not changed (gh-9195).
```

Comment:
We started to log unchanged options in commit ecdf3c532d628, which was merged to 3.0.0-entrypoint-198-gecdf3c532d62, which hasn't been released yet. So it looks like we don't need a changelog.

Reply:
Thanks! Dropped the changelog.